src/backend/communities.nim

```diff
@@ -433,12 +433,24 @@ proc collectCommunityMetrics*(communityId: string, metricsType: int, intervals:
       "intervals": intervals
     }])
 
-proc requestCommunityInfo*(communityId: string, tryDatabase: bool): RpcResponse[JsonNode] {.raises: [Exception].} =
-  result = callPrivateRPC("fetchCommunity".prefix, %*[{
-    "communityKey": communityId,
-    "tryDatabase": tryDatabase,
-    "waitForResponse": true
-  }])
+proc requestCommunityInfo*(communityId: string, tryDatabase: bool, shardCluster: int, shardIndex: int): RpcResponse[JsonNode] {.raises: [Exception].} =
```

Comment:
I tried. First, since I use an async_task, I still needed to recreate the `Shard` object since I can only pass strings and ints to the threads. Also it crashed so I said fuck it lol